Source/Mesh/MeshBoundaryMethods.f90

```diff
@@ -931,7 +918,6 @@ SUBROUTINE AssociateBoundaryEdgesToCurve( list, chain, isInnerBoundaryCurve )
             p    = node % x
             dMin = HUGE(dMin)
             DO k = 0, totalCurvePoints - 1
-!               d = SQRT( distanceSquaredBetweenPoints(xCurve(:,k),p,nHat(:,j)) )
                d = SQRT( distanceSquaredBetweenPoints(xCurve(:,k),p,[0.0_RP,0.0_RP,0.0_RP]) )
```

Comment:
The nHat was put in to allow finding the minimum distance in a given direction, but it didn't pan out, i.e. it didn't seem to turn out to be helpful. It could be removed, but my thought is just to leave it in on the chance that we may want to use it later.

Reply:
Sure, that is fine by me so we would not need to re-implement it in the future. But we could  comment it out and put a comment block above it that states basically what you wrote above. Just to save some memory and unnecessary computation.

Reply:
Will do.